package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.3",
   "description": "Javascript module template.",
   "main": "index.js",
```

Comment:
Thanks for the merge - might also make sense to update the `main` entry here as it's pointing at the non-functional `index.js` (that's pointed at `./dist/xcomponent-demo`).  Should probably also point at "dist/brainblocks.js"